### PR TITLE
Add batch delete index

### PR DESCRIPTION
**Pull request by xiaojun207**
Add batch delete index

### CODE REVIEW

File: pkg/handlers/index/delete.go [hunk 1]
```diff
@@ -27,32 +28,36 @@ import (
...
 		return
 	}
 
-	// delete meta
-	if err := core.DeleteIndex(index.Name); err != nil {
-		c.JSON(http.StatusInternalServerError, meta.HTTPResponseError{Error: err.Error()})
-		return
+	for _, indexName := range strings.Split(indexNames, ",") {
+		index, exists := core.GetIndex(indexName)
+		if !exists {
```
Comment (oasisk) [inline pkg/handlers/index/delete.go:48]: As part of core.DeleteIndex we check if index exists , the check for index exists can be removed from line 47

File: web/src/views/Index.vue [hunk 2]
```diff
@@ -221,7 +235,38 @@ export default defineComponent({
...
+    };
+
+    const deleteSelectedIndexes = () => {
+      if (!selectedIndexes || selectedIndexes.value.length == 0) {
+        $q.notify({
+          position: "top",
+          color: "warning",
+          textColor: "white",
+          icon: "warning",
+          message: "Please select index for delete",
```
Comment (oasisk) [inline web/src/views/Index.vue:250]: request you to change it to "Please select index for deletion"

**Review by oasisk — APPROVED** [review]

File: web/src/views/Index.vue (hunk 2 above)
Comment (xiaojun207) [reply web/src/views/Index.vue:250]: ok，thank

**Review by hengfeiyang — COMMENTED** [review]
update swager docs

**Review by hengfeiyang — APPROVED** [review]

**Pull request merged**